source/components/longClickButton/longClickButton.html

```diff
@@ -1,11 +1,10 @@
-<span class="long-click-button">
-	<button id="actionButton" class="btn {{button.types}} {{button.configuredSize}}" ng-mousedown="button.startAction()" ng-mouseleave="button.stopAction()" ng-mouseup="button.stopAction()" ng-disabled="button.busy || button.ngDisabled">
-		<rl-busy loading="button.busy" ng-if="button.rightAligned"></rl-busy>
-		<span ng-transclude>
-			<i ng-show="button.icon != null" class="fa fa-{{button.icon}}"></i> {{button.buttonText}}
-		</span>
-		<rl-busy loading="button.busy" ng-if="!button.rightAligned"></rl-busy>
-	</button>
-	<rl-rating-bar ng-if="button.active" width="button.width" height="5" min="0" max="button.duration"
-				value="button.actionProgress" background="transparent"></rl-rating-bar>
-</span>
\ No newline at end of file
+<button id="actionButton" class="btn btn-long-click {{button.types}} {{button.configuredSize}}"
```

Comment:
This is when our component based architecture gets to show its paces. How cool is it that we can change this here in one place and have it update everywhere in the app without touching tons of files... !